app/views/search/_country.html.erb

```diff
@@ -7,6 +7,7 @@
   </h3>
 
   <p>
-    <%= StatisticPresenter.new(model).percentage_pa_cover %>% covered by Protected Areas
+    <% stats = StatisticPresenter.new(model) %>
+    <%= (stats.percentage_pa_land_cover + stats.percentage_pa_marine_cover).round(2) %> % covered by Protected Areas
```

Comment:
Could you please add this logic into the Presenter module?
It would be good to keep the views agnostic in terms of any logic, even for a simple one like the sum

Reply:
Sure @Levia great point!